CMakeLists.txt

```diff
@@ -401,42 +426,42 @@ endif()
 # rc files are only used on Windows binaries
 if(WIN32)
   set(MagickLibVersionNumber ${MAGICK_LIB_VERSION_NUMBER})
-  set(MagickLibVersionText \"${MAGICK_LIB_VERSION_TEXT}\")
-  
+  set(MagickLibVersionText ${MAGICK_LIB_VERSION_TEXT})
+
   if(${CMAKE_SIZEOF_VOID_P} EQUAL 4)
     set(MagickPlatform "x86")
   elseif(${CMAKE_SIZEOF_VOID_P} EQUAL 8)
     set(MagickPlatform "x64")
   else()
     set(MagickPlatform ${MAGICK_TARGET_CPU})
   endif()
-  
+
   # Read the version.h file so we can get some needed values using regex
   file(READ ${CMAKE_CURRENT_BINARY_DIR}/MagickCore/version.h CONTENT_VERSION_H)
-  
+
   # Try to find `MagickCopyright'
-  string(REGEX MATCH "\n#define[ ]+MagickCopyright[ ]+\"([^\"]+)\"\n" FOUND_MAGICK_COPYRIGHT ${CONTENT_VERSION_H})
+  string(REGEX MATCH "\n#define +MagickCopyright +\"([^\"]+)\"\n" FOUND_MAGICK_COPYRIGHT ${CONTENT_VERSION_H})
   if(FOUND_MAGICK_COPYRIGHT)
     set(MagickCopyright \"${CMAKE_MATCH_1}\")
```

Comment:
Remove quotes here too? All the quote removals break the ImageMagick.rc.cmake, but I can fix that if you want.

Reply:
Oops, I have not yet tested my changes on Windows. I definitely should.